### PR TITLE
Fixes #28981 - Add description to plugin roles

### PR DESCRIPTION
**Pull request by stejskalleos**
PR is part of  [Add default descriptions to all roles added from plugins](https://projects.theforeman.org/issues/28936) task.

### CODE REVIEW

**Comment by stejskalleos** [comment]
@lzap Can you please give me description for the roles? Thanks

File: lib/foreman_discovery/engine.rb [hunk 1]
```diff
@@ -148,8 +148,8 @@ class Engine < ::Rails::Engine
           :edit_discovery_rules,
           :destroy_discovery_rules,
         ]
-        role "Discovery Reader", READER
-        role "Discovery Manager", MANAGER
+        role "Discovery Reader", READER, "Role granting permissions to view discovery plugin features."
```
Comment (lzap) [inline lib/foreman_discovery/engine.rb]: Role granting permissions to view discovered hosts

File: lib/foreman_discovery/engine.rb [hunk 2]
```diff
@@ -148,8 +148,8 @@ class Engine < ::Rails::Engine
...
+        role "Discovery Manager", MANAGER, "Role granting permissions to manage discovery plugin."
```
Comment (lzap) [inline lib/foreman_discovery/engine.rb]: Role granting permissions to perform provisioning of discovered hosts

**Comment by stejskalleos** [comment]
[test foreman_discovery]

**Review by lzap — APPROVED** [review]
Thanks a bunch.

**Pull request merged**